data-import-from-s3/src/test/java/no/unit/nva/dataimport/StubS3Driver.java

```diff
@@ -40,15 +41,22 @@ public String getFile(String filename) {
     }
 
     private Stream<String> fileContent(String filename) {
-        InputStream inputStream = IoUtils.inputStreamFromResources(filename);
-        if (streamIsEmpty(inputStream)) {
-            throw new IllegalStateException("File does not exist or file is empty");
-        }
+        InputStream inputStream = attempt(() -> IoUtils.inputStreamFromResources(filename))
+                                      .orElseThrow(fail -> fileNotFoundException());
+
         GZIPInputStream gzipInputStream = attempt(() -> new GZIPInputStream(inputStream)).orElseThrow();
         BufferedReader reader = new BufferedReader(new InputStreamReader(gzipInputStream));
```

Comment:
Maybe force charset onto the inputstreamreader also?  
 BufferedReader reader = new BufferedReader(new InputStreamReader(gzipInputStream , StandardCharsets.UTF_8));